notes/src/components/List.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import ListCard from './ListCard';
+
+export default class Movie extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            List: null
+        };
+    }
+
+    componentDidMount() {
+        // change this line to grab the id passed on the URL
+        const { id } = this.props.match.params;
+        this.fetchList(id);
+    }
+
+    fetchMovie = id => {
+        axios
+            .get(`http://localhost:3000/api//${id}ListCard`)
+            .then(response => {
+                this.setState(() => ({ List: response.data }));
+            })
+            .catch(error => {
+                console.error(error);
+            });
+    };
+    // Uncomment this code when you're ready for the stretch problems
```

Comment:
It looks like you've copied and pasted from previous assignments. Please don't do this.